drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_umac_if.h

```diff
@@ -3212,6 +3212,11 @@ struct nrf_wifi_umac_cmd_config_extended_ps {
 #define NRF_WIFI_MAX_TWT_FLOWS 8
 #define NRF_WIFI_PS_MODE_LEGACY 0
 #define NRF_WIFI_PS_MODE_WMM 1
+/* As per hardware design for power efficiency
```

Comment:
I don't think this is has to be with hardware design, but I might be wrong, can we elaborate on this? The comment should explain why 3, it now just explains that it is 3.